homeassistant/components/sleepiq/manifest.json

```diff
@@ -11,5 +11,5 @@
   "documentation": "https://www.home-assistant.io/integrations/sleepiq",
   "iot_class": "cloud_polling",
   "loggers": ["asyncsleepiq"],
-  "requirements": ["asyncsleepiq==1.5.2"]
+  "requirements": ["asyncsleepiq==1.5.3"]
```

Comment:
Please do the library bump in a separate PR unless the library bump requires the changes in this PR.